core/src/main/java/org/mskcc/cbio/portal/scripts/ImportProfileData.java

```diff
@@ -100,7 +100,7 @@ public void run() {
                 importer.importData();
             } else if (geneticProfile.getGeneticAlterationType() == GeneticAlterationType.GENERIC_ASSAY) {
                 // add all missing `genetic_entities` for this assay to the database
-                ImportGenericAssayEntity.importData(dataFile, geneticProfile.getGeneticAlterationType(), geneticProfile.getOtherMetaDataField("generic_entity_meta_properties"));
+                ImportGenericAssayEntity.importData(dataFile, geneticProfile.getGeneticAlterationType(), geneticProfile.getOtherMetaDataField("generic_entity_meta_properties"), true);
```

Comment:
Why do we always overwrite the entities when importing profiles? Should this not be linked to the CLI variable?